sql/catalyst/src/main/scala/org/apache/spark/sql/internal/SQLConf.scala

```diff
@@ -2595,6 +2595,15 @@ object SQLConf {
       .checkValue(_ > 0, "The timeout value must be positive")
       .createWithDefault(10L)
 
+  val LEGACY_ALLOW_CAST_NUMERIC_TO_TIMESTAMP =
+    buildConf("spark.sql.legacy.allowCastNumericToTimestamp")
+      .internal()
+      .doc("When true, allow cast numeric to timestamp, but for integral numbers," +
+        "Hive treats it as milliseconds, Spark SQL treats n as seconds")
```

Comment:
Do we need to mention hive here ? Can we not just state the spark behavior ? Secondly not sure what is N here ?